CHANGELOG.rst

```diff
@@ -1,40 +1,49 @@
 2.9.0.dev
 =========
 
-New Features
-------------
+**New Features**
```

Comment:
personally i#d prefer titles/subtitles - are there ways to manually override the target name perhaps?